source/GameAction.cpp

```diff
@@ -127,6 +127,23 @@ namespace {
 			message += "flagship.";
 		Messages::Add(message, Messages::Importance::High);
 	}
+
+	double OutfitBulkBonus(int count)
+	{
+		if(count < 50)
+			return 1.05;
```

Comment:
Just a question, but what are these values based on?

Reply:
Just something I came up with. Honestly I am not even sure if I like the bulk bonus idea or not.

Reply:
It makes sense to me, maybe you can implement it, so that it can be changed from mission files, idk

Reply:
Yeah that would make more sense. It could just be defined up front and then this logic can be removed. Let me think about how to best fit that in.